builders/ethereum/json-rpc/moonbeam-custom-api.md

```diff
@@ -58,3 +63,30 @@ To begin exploring Moonbeam's custom JSON-RPC endpoints, you can try out the pro
           "params": ["INSERT_TRANSACTION_HASH"]
         }' {{ networks.moonbase.rpc_url }}
         ```
+
+???+ function "moon_getLatestSyncedBlock"
+
+    Returns the latest synced block number.
+
+    === "Parameters"
+
+        None
+
+    === "Returns"
```

Comment:
Also want to mention the data type that is being returned, in this case, an integer